cfgov/jinja2/v1/_includes/organisms/omwi-salesforce-outage-banner.html

```diff
@@ -19,12 +19,11 @@
         {{ notification.render(
             'warning',
             true,
-            'We are currently experiencing technical difficulties with the
-             Office of Minority and Women Inclusion (OMWI) assessment form
-             and inclusivity portal.',
-            'We are actively working on the issue, so please check back later
-             if you need to submit an OMWI assessment or
-             check the status of one you have already submitted.'
+            'We are experiencing technical difficulties with the Office of
+             Minority and Women Inclusion (OMWI) assessment form and inclusivity
+            portal.',
```

Comment:
This is so trivial, but it'll bug me if I don't point it out 😆 

```suggestion
             portal.',
```